test/test.js

```diff
@@ -213,9 +213,41 @@ describe('paste-markdown', function () {
       paste(textarea, {'text/html': sentence, 'text/plain': plaintextSentence})
       assert.equal(textarea.value, markdownSentence)
     })
+
+    // Note: It's possible to construct and dispatch a synthetic paste event,
+    // but this will not affect the document's contents in tests to assert it.
```

Comment:
```suggestion
    // but this will not affect the document's contents in tests to assert it.
    // https://developer.mozilla.org/en-US/docs/Web/API/Element/paste_event
```